### PR TITLE
Music for the bar

### PR DESCRIPTION
**Pull request by TheGoret**
-Removed the candles box as there is already two in the backroom, replacing it with cigars. 
-Removed the cake hat, as it doesn't really make sense to have it here and nobody use it.
-Added a phonograph, as it add to the bar ambiance and coziness.

### CODE REVIEW

**Comment by TheGoret** [comment]
!review

**Comment by alsoandanswer** [comment]
>Removed the candles box as there is already two in the backroom, replacing it with cigars.
the candles left on the outside so people **other** than the bartender can re-stock the tables with candles

>Removed the cake hat, as it doesn't really make sense to have it here and nobody use it.
the cake hat HAS seen use, and it is not in the loadout, so removing it will make it virtually unobtainable. at least move it to the kitchen, or the stage area.

**Comment by TheGoret** [comment]
Moved the cake-hat to the kitchen and re-added a candles pack.

File: html/changelogs/goret-music-for-the-bar.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "A phonograph and a cigar case have been added to the bar counter. The cake hat has been moved to the kitchen."
```
Comment (furrycactus) [inline html/changelogs/goret-music-for-the-bar.yml]: Change "rscadd" to "maptweak" maybe. Not vital.

Comment (TheGoret) [reply html/changelogs/goret-music-for-the-bar.yml]: Done! Thx.

**Pull request merged**